pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -377,7 +377,7 @@ func (s *Summarizer) loadVersionMapping() error {
 	}
 
 	kubeToBenchmarkMap := v.GetStringMapString(VersionMappingKey)
-	if kubeToBenchmarkMap == nil || (len(kubeToBenchmarkMap) == 0) {
```

Comment:
Removed `kubeToBenchmarkMap == nil` as that is redudant, `len` also does such checks.